### PR TITLE
bump deps 2024

### PR DESCRIPTION
**Pull request by odsod**
- chore: update CODEOWNERS
- feat: bump deps 2024

### CODE REVIEW

File: .sage/main.go [hunk 1]
```diff
@@ -28,13 +27,13 @@ func main() {
...
 func All(ctx context.Context) error {
-	sg.Deps(ctx, ConvcoCheck, GoLint, GoReview, GoTest, FormatMarkdown, FormatYAML)
+	sg.Deps(ctx, ConvcoCheck, GoLint, GoTest, FormatMarkdown, FormatYaml)
 	sg.SerialDeps(ctx, Proto.All, TypescriptLint)
 	sg.SerialDeps(ctx, GoModTidy, GitVerifyNoDiff)
 	return nil
 }
 
-func FormatYAML(ctx context.Context) error {
+func FormatYaml(ctx context.Context) error {
```
Comment (alethenorio) [inline .sage/main.go:36]: YAML is an acronym. The general Go [guideline](https://go.dev/wiki/CodeReviewComments#initialisms) is to make it uppercase.

Comment (odsod) [reply .sage/main.go:36]: copy pasted this from the Sage example: https://github.com/einride/sage/blob/master/example/.sage/main.go#L72 - we typically break convention here to make the generated kebab-case always consistent

Comment (alethenorio) [reply .sage/main.go:36]: The kebab case should be correctly generated (as noted by the fact the makefile target hasn't changed) so why so we want to break convention? Maybe we can instead fix the Sage target?

Comment (odsod) [reply .sage/main.go]: sounds good, updated

**Review by alethenorio — APPROVED** [review]

**Pull request merged**